stag_detect:
  ros__parameters:

    # STag parameters
    stag_library: 11
    error_correction: 2

    # Camera configuration
    camera_info_topic: camera/camera_info
    image_topic: camera/image_raw
    is_compressed: false

    # Node configuration
    tag_tf_prefix: fiducial_
    show_markers: true
    publish_tf: true
